From ebb63d2100983e34f837f11d02947042e179a769 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 25 Apr 2018 10:57:36 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: enable BIFROST Mali for
 linux

In order to support the rk3326 mali on linux platform, enabling the
BIFROST Mali config for linux.

As the mali driver had supported for linux with the below commits[0].
And the linux platform didn't need to loade the mali module for working.

commits[0]:
2aee160 MALI: bifrost: RK:
    There are a few modifications in some 'Kbuild' and 'Kconfig' files.
e5ccb3a MALI: bifrost: RK:
    add separate src dir of Bifrost driver for RK Linux device

Change-Id: I1cbd8515aab8a7bd23ab31db1743a4b8c723d81d
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 71c17313c1a7..2c2dae770e72 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -328,7 +328,8 @@ CONFIG_MALI_EXPERT=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
-CONFIG_MALI_BIFROST=m
+CONFIG_MALI_BIFROST_FOR_LINUX=y
+CONFIG_MALI_BIFROST=y
 CONFIG_MALI_BIFROST_DEVFREQ=y
 CONFIG_MALI_PLATFORM_NAME="rk"
 CONFIG_MALI_BIFROST_EXPERT=y
-- 
2.35.3

